README.md

```diff
@@ -7,39 +7,37 @@
 
 ## Purpose
 
-$${\textsf{\color{#1ab458}Discussion closure should rest with the initiator.}}$$
```

Comment:
LaTeX doesn't get rendered properly in Markdown, so I switched to HTML

Reply:
Not on phone but in browser it's better, in GitHub Markdown, style tags etc. are stripped for safety reasons, this coloring etc is not there. It's recommended to use LaTeX

Reply:
Okay, makes sense 👍🏻 
You are right - It didn't work on my phone, in GH app, but it didn't work in my IDE interpreter either.

But I am okay with LaTeX :)

Reply:
Works fine in WebStorm :)